### ISSUE TITLE
`cargo codspeed build` doesn't build virtual manifest

### ISSUE THREAD

**Issue by tversteeg** [CodSpeedHQ/codspeed-rust#37]
I'm trying to integrate CodSpeed into [this](https://github.com/tversteeg/blit) repo which contains a virtual manifest. So the `Cargo.toml` contains a `[workspace]` section but the main package is not included in it.

When I run `cargo codspeed build` I get the following output:

```
   Collected 1 benchmark suite(s) to build
    Building benchmarks
       Error There are multiple `blit` packages in your project, and the specification `blit` is ambiguous.
Please re-run this command with `-p <spec>` where `<spec>` is one of the following:
  file:///.../blit#0.8.5
  https://github.com/rust-lang/crates.io-index#blit@0.8.5
```

There's no way to select the main package, which should be selected by default anyway.

**Comment by adriencaccia** [comment]
Hey @tversteeg, thanks for the issue!

I managed to reproduce the error investigating with your repository.

The root cause is not due to the virtual manifest, but rather tied to the fact that `blit` is dependent on itself, because of [its dependency `pixel-game-lib`](https://github.com/tversteeg/blit/blob/1403e12f924af7158350eac621dfbbbe9116961c/Cargo.toml#L24) which has `blit` as a dependency.

I submitted a PR with a fix.
